UKHomeOffice/sgmr-service
# SPCR-414 Add loading component to all forms

### AC / Description of changes
We have to cater to slow networks so need the loading component to ensure page doesn’t load with previous/missing data

Definitely needed on
- new people forms
- new pleasure craft forms

Check voyage and Onboarding if they are missing it too

### To Test
- Inspect and click on network tab
- Swap throttling to slow 3G
- Complete a voyage plan up until the CYA page
- > When a new form loads there should be a loading spinner
- When on the CYA page, click on the change links
- > A loading spinner should appear and the page should load with the inputted data visible
- Click on the dashboard tab
- > A loading spinner should be visible before the page loads
- > Go to the people tab 
- Click on update a person
- > The loading spinner should be visible before the form appears with the pre-filled person data
- > This should be the same with the PC page

### Notes

- There is no loading spinner n the PC tab due to the way the page is set up using page container
- Onboarding did not need loading forms as the forms loaded no problem on slow 3G
- Changed wording on update column of peopleTable to remove duplicate word and prevent tests from failing

---

**Remember to merge to feature branches not `master`.**
